internal/wireguard/rule_test.go

```diff
@@ -83,7 +83,7 @@ func Test_Wireguard_addRule(t *testing.T) {
 
 			netLinker.EXPECT().RuleAdd(testCase.expectedRule).
 				Return(testCase.ruleAddErr)
-			cleanup, err := wg.addRule(rulePriority, firewallMark)
+			cleanup, err := wg.addRule4(rulePriority, firewallMark)
```

Comment:
Rename test to `Test_Wireguard_addRule4`.

Also a test for addRule6 would be nice